src/pages/NotFoundPage/NotFound.tsx

```diff
@@ -16,11 +13,6 @@ const NotFound: React.FC = () => {
   }
   return (
     <EmptyState variant={EmptyStateVariant.lg}>
-      <EmptyStateHeader
-        titleText="Your organization currently has no subscriptions for this product"
-        icon={<EmptyStateIcon icon={WrenchIcon} color="grey" />}
-        headingLevel="h5"
-      />
```

Comment:
Issue: This content still needs to be shown somehow - if the header component is deprecated there needs to be some other way to render this